src/tests/components/ForecastDetails.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("Forecast Details", () => {
+  const validProps = {
+    date: 2222222,
+    humidity: 80,
+    wind: {
+      speed: 30,
+      direction: "s",
+    },
+    temperature: {
+      max: 33,
+      min: 22,
+    },
+  };
+
+  it("Renders Forecast Details", () => {
+    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders the correct element", () => {
+    const { getByTestId } = render(<ForecastDetails forecast={validProps} />);
+    const ancestor = getByTestId("ancestor");
+    const descendant = getByTestId("descendant");
```

Comment:
Is it really important that a given component is child of another? Also, does this test validate anything about what is actually displayed on the screen? 
Remember when approaching react testing, the primary concern is what is displayed to the user and their experience of it. Would it not be better to assert that given bits of text are displayed instead? This would also mean we could remove the getByTestId query, which we want to avoid if at all possible